### ISSUE TITLE
toolkit build fails in Streams QSE

### ISSUE THREAD

**Issue by RolefH** [IBMStreams/streamsx.messagehub#46]
When building the toolkit with Streams Quickstart Edition, build fails:
```
[streamsadmin@streamsqse com.ibm.streamsx.messagehub]$ ../gradlew build
1.3.1
:com.ibm.streamsx.messagehub:getDeps

FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all dependencies for configuration ‘:com.ibm.streamsx.messagehub:runtime’.
> Could not resolve IBMStreams:streamsx.kafka:v1.3.1.
 Required by:
     project :com.ibm.streamsx.messagehub
  > Could not resolve IBMStreams:streamsx.kafka:v1.3.1.
     > Could not get resource ’https://github.com/IBMStreams/streamsx.kafka/archive/v1.3.1.xml'.
        > Could not GET ‘https://github.com/IBMStreams/streamsx.kafka/archive/v1.3.1.xml’.
           > Received fatal alert: protocol_version
  > Could not resolve IBMStreams:streamsx.kafka:v1.3.1.
     > Could not get resource ‘https://github.com/IBMStreams/streamsx.kafka/archive/v1.3.1.xml’.
        > Could not GET ‘https://github.com/IBMStreams/streamsx.kafka/archive/v1.3.1.xml’.
           > Received fatal alert: protocol_version

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

**Comment by RolefH** [comment]
The messages are a little bit misleading, but the root cause is that Github does not support TLS v1.0 anymore, and IBM Java offers only TLS v1.0 as default.

Workarounds:
a) use Oracle Java or openJDK to build, or
b) add a property file `gradle.properties` to the root directory of the repo with following content:
```
org.gradle.jvmargs="-Dcom.ibm.jsse2.overrideDefaultTLS=true"
```

**Comment by RolefH** [comment]
corrected in v1.3.2
